boards/arm/efr32_radio/doc/brd4250b.rst

```diff
@@ -0,0 +1,116 @@
+.. _efr32_radio_brd4250b:
+
+EFR32 BRD4250B (SLWRB4250B)
+###########################
+
+Overview
+********
+
+The EFR32FG1 2400/868 MHz Dual Band Radio Board is delivered as part of
```

Comment:
Could be add an image of the daugher board here? This would easily show what this page is about (same case for the EFR32BG page).